jdbc/core/src/test/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURLManagerTest.java

```diff
@@ -46,4 +58,14 @@ public void assertToAbsolutePathConfigurationFile() {
         byte[] actual = ShardingSphereDriverURLManager.getContent("jdbc:shardingsphere:absolutepath:" + absolutePath);
         assertThat(actual.length, is(fooDriverConfigLength));
     }
+    
+    @org.junit.jupiter.api.Test
```

Comment:
Why not use `org.junit.Test` here?

Reply:
Because of using junit5 `MockStatic`, it is the new API.